src/components/Components/ProjectDetailsPanelComponent/ProjectDetailsPanelComponent.tsx

```diff
@@ -0,0 +1,34 @@
+import { SVGAtom, type IconNames } from "~/components/Atoms/SVGAtom/SVGAtom";
+
+export type ItemIcon = {
```

Comment:
I usually don't use Array type mainly, but I declare array type in T[]; form.
So I don't know what advantages a declaration in the form of Array has, can you tell me the advantages of the declaration Ekan used?

If it doesn't have a big advantage about declaring a type in the form T[] seems the easiest, so I'm going to unify it in this form. Please feel free to let me know what Ekan thinks